futurex_openedx_extensions/helpers/tasks_utils.py

```diff
@@ -97,15 +98,15 @@ def _paginated_response_generator(
         url = response.data.get('next')
 
 
-def _upload_file_to_storage(local_file_path: str, filename: str) -> str:
+def _upload_file_to_storage(local_file_path: str, filename: str, tenant: str) -> str:
     """
     Upload a file to the default storage (e.g., S3).
 
     :param local_file_path: Path to the local file to upload
     :param filename: ilename for generated CSV
     :return: The path of the uploaded file
     """
-    storage_path = os.path.join(settings.FX_DATA_EXPORT_DIR_NAME, filename)
+    storage_path = os.path.join(settings.FX_DATA_EXPORT_DIR_NAME, tenant, filename)
```

Comment:
Sorry if it wasn't clear. Please change the default of `FX_DATA_EXPORT_DIR_NAME` to 

```python
    settings.FX_DATA_EXPORT_DIR_NAME = getattr(
        settings,
        'FX_DATA_EXPORT_DIR_NAME ',
        'exported_files'
    )
```

and create a function to return the directory, something like
```
def get_dir_of_storage(tenant_id: int, dir: str):
    return os.path.join(f'fx_dashboard/{tenant_id}', dir)
```

so here:
```suggestion
    storage_path = get_dir_of_storage(tenant_id, os.path.join(settings.FX_DATA_EXPORT_DIR_NAME, filename))
```

Reply:
done